Formula/gst-python.rb

```diff
@@ -10,21 +10,50 @@ class GstPython < Formula
     sha256 "e708aa675d5744fc36608249728290d6d97803cd071bf2ac91d245a9d8e8c996" => :mavericks
   end
 
+  option "without-python"
+
+  depends_on :python3 => :optional
   depends_on "gst-plugins-base"
-  depends_on "pygobject3"
+
+  requires_py3 = []
+  requires_py3 << "with-python3" if build.with? "python3"
+  depends_on "pygobject3" => requires_py3
```

Comment:
I think it's clearer (and more common, see e.g. the `pyqt`, `pyside`, `qscintilla2`, etc. formulae) to make that:

```ruby
if build.with? "python3"
  depends_on "pygobject3" => "with-python3"
else
  depends_on "pygobject3"
end
```